cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -437,6 +437,20 @@ spec:
             - name: EMIT_KUBE_NODE_STATUS_CAPACITY_CPU_CORES_METRIC
               value: "true"
             {{- end }}
+            {{- if .Values.kubecostMetrics }}
```

Comment:
no objections here, but could/should this be made more general? 

e.g. EMIT_KUBE_LABELS_METRIC or EMIT_KSM_METRICS

just trying to help avoid future config overload if possible

Reply:
We could change to something like `EMIT_KSM_V1_METRICS`, which I think is a pretty good idea. I doubt there will be a situation where we want to emit only some of them. I'd like a third opinion, @AjayTripathy / @mbolt35 / @nikovacevic

Reply:
Perhaps `EMIT_NECESSARY_KSM_V1_METRICS`

Reply:
`EMIT_KSM_V1_METRICS` is good with me. Seems like we might need a broader discussion around emitting metrics this way, as it could quickly become messy no matter what name we pick.

Reply:
Yes, agreed. I'll get the ball rolling for `EMIT_KSM_V1_METRICS`. Should we discuss this week?

Reply:
https://github.com/kubecost/cost-model/pull/833